ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,116 @@
+### Diamond Weapon Ex
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# -ii 5FEB 5FBE 5FAA 5FEC 5FBA 5FFD 5FBB 6055 6159 5FF9 5FCD 5FFA 5FF9 612E 5FB3 5FCE 5FD2 5FFE 5FD3 5FB4 5FB6 5FB1 5FB0 5FBF 5FAE 5FA6 5FC3 5FC9 5FCA 5FC4
+
+### Phase 1
+# There are no autoattacks from the boss until after the initial raidwide, so nothing to sync against for autos
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
+
+# Adamant Purge is a mini-phase that will be either stack, spread, or "nothing"
+# 5FA2 = Spread (Homing Laser, 5FA0)
+# 5F9B = "nothing" (Diamond Rain, 5FA7)
+# 5FA5 = Stack (Diamond Flash, 5FA1)
+# This mechanic repeats in phase 3 with additional bits and an extra tracking If circle aoe
+# @TODO: Someone better at timelines than me, sync these to mini-phases instead
+36.5 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/ window 20,5
+46.7 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+49.9 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+67.1 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+83.4 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/
+93.6 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+96.8 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
```

Comment:
Is this true? I think it's always claw 1, spread/stack, claw 2, nothing, claw 3, stack/spread.  I do not see such a 5FA[01] in my logs.  e.g. I see a 5FA3/5FA0, a 5F9A on its own, then a 5FA5/5FA1.  Do you have a counterexample log?

Reply:
can confirm i expirienced always the same that 2. claw doe snot havespread/stack but it has the orbs (where the grous need to be split up on the sides)

Reply:
The Adamant Purge for Diamond Rain can be either 5F9A or 5F9B depending on cleave side (see the triggers PR), but otherwise now that I can more easily check the actual mechs without manually matching up events with movements, I can confirm that across about 50 pulls it's always stack/spread, then aoe, then stack/spread in both phases. I have updated the timeline to reflect this.